src/apis/setAnnotationContentOverlayHandler.js

```diff
@@ -0,0 +1,22 @@
+import action from 'actions';
+
+/**
+ * Adds a custom overlay to annotations if that annotation currently support it. Currently only the Ellipsis annotation supports it.
```

Comment:
Hmm maybe we forgot to change the description? ( •̀ .̫ •́ )✧

Reply:
My bad!